src/main/resources/templates/laDigitalAssister/howThisWorks.html

```diff
@@ -69,8 +69,7 @@
         <th:block th:replace="~{'fragments/honeycrisp/reveal' :: reveal(
                 controlId='r4',
                 linkLabel=~{::revealLabel4},
-                content=~{::revealContent4},
-                forceShowContent='true')}">
+                content=~{::revealContent4})}">
```

Comment:
Was this a design change request? It doesnt match Figma 
<img width="120" alt="Screen Shot 2024-01-26 at 12 40 28 PM" src="https://github.com/codeforamerica/la-doc-uploader/assets/72890349/1a0a4bc3-6557-441a-a2e5-943ede51aa31">